django/contrib/gis/admin/options.py

```diff
@@ -44,6 +47,14 @@ class GeoModelAdmin(ModelAdmin):
     debug = False
     widget = OpenLayersWidget
 
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            'django.contrib.gis.admin.GeoModelAdmin is deprecated '
+            'in favor of django.contrib.gis.admin.GISModelAdmin.',
```

Comment:
We could add a note about `OSMGeoAdmin`, e.g.
```suggestion
            'django.contrib.gis.admin.GeoModelAdmin and OSMGeoAdmin are '
            'deprecated in favor of django.contrib.gis.admin.GISModelAdmin.',
```